controller/RestDelivery.php

```diff
@@ -212,6 +214,50 @@ public function updateDeferred()
         }
     }
 
```

Comment:
There are some errors in swagger editor when I copy&paste the json. Can you fix them to "application/x-www-form-urlencoded"? Thanks!

`Semantic error at paths./taoDeliveryRdf/RestDelivery/generate.post
Operations with Parameters of "in: formData" must include "application/x-www-form-urlencoded" or "multipart/form-data" in their "consumes" property
Jump to line 14
`